docs/versionSelector.js

```diff
@@ -0,0 +1,69 @@
+import {
+  LitElement,
+  css,
+  html,
+} from "https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js";
+
+class VersionSelector extends LitElement {
+  static styles = css`
+    select {
+      padding: 5px;
+      margin: 10px 0;
+      font-size: 100%;
+    }
+  `;
+
+  static properties = {
+    versions: { type: Array },
+  };
+
+  constructor() {
+    super();
+    this.versions = [];
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.fetchVersions();
+  }
+
+  async fetchVersions() {
+    try {
+      const response = await fetch(
+        "https://martinvonz.github.io/jj/versions.json",
```

Comment:
This file is currently generated by mike. We'd have to find another way to update it. Luckily the format is super simple:

```json
[
  {
    "version": "prerelease",
    "title": "prerelease",
    "aliases": []
  },
  {
    "version": "v0.23.0",
    "title": "v0.23.0",
    "aliases": [
      "latest"
    ]
  },
  {
    "version": "v0.22.0",
    "title": "v0.22.0",
    "aliases": []
  },
  {
    "version": "v0.21.0",
    "title": "v0.21.0",
    "aliases": []
  },
  ...
]
```

Reply:
I think we'd need a replacement for `mike`, yes.

This is my main worry about this PR: for a moment, I thought that `mkdocs` finally implemented a version switcher, but it seems that we'd have to take care of its guts ourselves if we go with mdbook. Perhaps they would accept such tools as part of their project?